bliss/simulator/decoder.py

```diff
@@ -98,6 +99,7 @@ def _get_fit_file_psf_params(psf_fit_file: str, bands: Tuple[int, ...]):
         Returns:
             psf_params: tensor of parameters for each band
         """
+        assert os.path.exists(psf_fit_file), f"{psf_fit_file} does not exist"
```

Comment:
nit: Should we suggest a fix in the error message, e.g. "ensure `fields` in field_list config correspond to available data files" or something along those lines?

Reply:
Updated error messages